src/App.js

```diff
@@ -1,7 +1,37 @@
-import "./App.css";
+import React from "react";
+import ItemList from "./Component/ItemList";
+import CheckOut from "./Component/CheckOut";
+import Cart from "./Component/Cart"
 
-const App = () => {
-  return <h1>Hello, world!</h1>;
-};
 
-export default App;
+
+class App extends React.Component {
+   state ={cart:[], subtotal:0}
```

Comment:
since the subtotal can be calculated based on the items in the shopping cart, subtotal should not be in state.